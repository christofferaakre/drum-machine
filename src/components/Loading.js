import React from 'react';

export const styles = {
  container: {
    display       : 'flex',
    alignItems    : 'center',
    justifyContent: 'center',
  },
};

export const Loading = () => (
  <div style={styles.container}>
    <img style={styles.image} alt='loading' src='images/loader.gif' />
  </div>
);

export default Loading;
