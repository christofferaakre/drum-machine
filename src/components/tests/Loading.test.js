import React from 'react';
import { shallow, } from 'enzyme';
import { Loading, } from '../Loading';

let props;

beforeEach(() => {
  props = {};
});

describe('Loading', () => {
  test('renders correctly', () => {
    const wrapper = shallow(<Loading {...props} />);

    expect(wrapper).toMatchSnapshot();
  });
});
