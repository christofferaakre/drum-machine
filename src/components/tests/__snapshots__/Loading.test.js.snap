// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "justifyContent": "center",
    }
  }
>
  <img
    alt="loading"
    src="images/loader.gif"
  />
</div>
`;
